### DISCUSSION TITLE
A bit of meta-discussion: Where best to ask for help?

### DISCUSSION THREAD

**Discussion by OranguTech** [pester/Pester#2091] [Q&A] [answered]
I don't want to clutter or bother the main devs if not necessary. Is this the best place for general questions? Lots of places listed on pester.dev:

StackOverflow is great for answering specific questions, not so much for general "I'm having trouble with X". So far I've found the Gitter instance is dead, Powershell.org forums nearly so. Some activity in /r/powershell/ (Reddit), but it's a crapshoot. Is the slack better? I tried joining but CBA to sort out that login/expired invite link yet.

**Comment by fflaten** [comment, marked as answer]
Discord and Slack are active and synced. Discord can be found here https://aka.ms/psdiscord .

**Reply by OranguTech** [reply]
Thanks! Note that the discord (and some sort of IRC bridge?) aren't listed on the community section of https://pester.dev/
